src/test/java/spoon/test/imports/ImportTest.java

```diff
@@ -1777,4 +1778,22 @@ public void testImportByJavaDoc() throws Exception {
 		}
 	}
 
+	@Test
+	public void testThatCorrectImportsAreGeneratedForJavaxAnnotatedElements() {
```

Comment:
In Spoon, the tests come with a one-line contract describing the test intention, something like

// contract: annotations are not added on import statements

could you add the contract that is tested? Thanks!